test/test-applications/integrations/TestApplication.EntityFrameworkCore.Pomelo.MySql/TestContext.cs

```diff
@@ -0,0 +1,38 @@
+// <copyright file="TestContext.cs" company="OpenTelemetry Authors">
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// </copyright>
+
+using Microsoft.EntityFrameworkCore;
+
+namespace TestApplication.EntityFrameworkCore.Pomelo.MySql;
+
+public class TestContext : DbContext
```

Comment:
```suggestion
public class DbTestContext : DbContext
```

Reply:
It was copy paste from the second EntityFrameworkCore project.

I can fix both places, but I preffer `TestDbContext`. If you are fine with current state, please just resolve comment.